terraform/aws/template_generator.go

```diff
@@ -31,7 +31,7 @@ func NewTemplateGenerator() TemplateGenerator {
 	}
 }
 
-func (tg TemplateGenerator) Generate(state storage.State) string {
+func (tg TemplateGenerator) Generate(state storage.State, _ bool) string {
```

Comment:
Could we make this work for all the IAASes? I'd like to not have this _only_ work for GCP.